socorro/external/es/crashstorage.py

```diff
@@ -34,24 +34,6 @@
 MAX_STRING_FIELD_VALUE_SIZE = 32_766
 
 
-class RawCrashRedactor(Redactor):
-    """Remove some specific keys from a dict. The dict is modified.
-
-    This is a special Redactor used on raw crashes before we send them
-    to our Elasticsearch database. It is used to remove fields that we don't
-    need to store, in order mostly to save some disk space and memory.
-
-    Not that this overwrites the list of forbidden_keys that would be defined
-    through configuration. That list is hard-coded in the __init__ function.
-    """
-
-    def __init__(self, config):
-        super().__init__(config)
-
-        # Overwrite the list of fields to redact away.
-        self.forbidden_keys = ["StackTraces"]
```

Comment:
We haven't had this key since before I started working on Socorro. So this class (as far as I can tell) has never had an effect on things.